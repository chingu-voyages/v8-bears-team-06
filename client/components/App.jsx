import React from "react";

class App extends React.Component {
  render() {
    return <div>Hello, you Chingu beauty!</div>;
  }
}

export default App;
