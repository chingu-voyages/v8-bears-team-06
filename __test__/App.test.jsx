import { shallow } from "enzyme";
import React from "react";
import renderer from "react-test-renderer";

import App from "../client/components/App";

describe("With Enzyme", () => {
  it('App shows "Hello, you Chingu beauty!"', () => {
    const app = shallow(<App />);

    expect(app.find("div").text()).toEqual("Hello, you Chingu beauty!");
  });
});

describe("With Snapshot Testing", () => {
  it('App shows "Hello, you Chingu beauty!"', () => {
    const component = renderer.create(<App />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
