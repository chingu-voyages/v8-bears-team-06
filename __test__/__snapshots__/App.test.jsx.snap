// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing App shows "Hello, you Chingu beauty!" 1`] = `
<div>
  Hello, you Chingu beauty!
</div>
`;
